org.eclipse.ui.console/src/org/eclipse/ui/console/AnsiConsolePageParticipant.java

```diff
@@ -0,0 +1,57 @@
+package org.eclipse.ui.console;
```

Comment:
This file requires a copyright header (see other files in the bundle).
also, if there is no need to have it public API, it's better to keep it in an internal package.